### PR TITLE
Grants Program review

### PR DESCRIPTION
**Pull request by mmagician**
Hey,

I made some small fixes:
- some are purely cosmetic, like typos & whitespaces
- there are two TODOs, which I would like you to address (either fix or comment on why not fixable/not applicable), as part of the review for [Milestone 1](https://github.com/w3f/Grant-Milestone-Delivery/pull/305/)

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -254,7 +256,12 @@ mod candle_auction {
                 }
             }
 
-            // pay the looser his bidded amount back
+            // pay the loser his bid amount back
+            // TODO: shouldn't we handle multiple accounts here? I guess the
```
Comment (agryaznov) [inline src/lib.rs]: Please see line  [415](https://github.com/agryaznov/candle-auction-ink/pull/1/files#diff-b1a35a68f14e696205874893c07fd24fdb88882b47c23cc0e0c80a30c7d53759L415): the payout is invoked by each participant by himself. Which is considered fair enough as winner payout presumably takes more gas fees to execute, hence everyone pays his own fees only

File: src/lib.rs [hunk 2]
```diff
@@ -303,19 +310,19 @@ mod candle_auction {
             }
         }
 
-        /// Pluggable reward logic: OPTION-1.
```
Comment (agryaznov) [inline src/lib.rs]: All these double-spaces in line endings are put there intentionally for formatting objectives: as in markdown syntax they force a newline (like `\n`). Just to make it look a little more prettier in final cargo doc. I'd like to leave them therefore (unless such a formatting violates any guidelines, does it?)

Comment (mmagician) [reply src/lib.rs]: Ah my bad! I wasn't aware of this formatting magic :) thanks for pointing it out

**Comment by agryaznov** [comment]
@mmagician thanks! 

this PR is partially accepted, providing comments explaining why proposed `TODOs` are `N\A`

**Pull request merged**